components/homepage/Header.tsx

```diff
@@ -0,0 +1,18 @@
+export function Header() {
+  return (
+    <>
+      <h1 className={"text-4xl font-bold text-center mb-5 title"}>Witaj!</h1>
+      <div className={"text-center"}>
+        <p className={"description"}>
+          Z racji na ukończenie przez Ciebie naszego planu stażowego,
+          chcielibyśmy poprosić Cię o wypełnienie ankiety na jego temat :)
+        </p>
+        <p className="description-mini">
```

Comment:
nit: You use two different conventions to write className values. I think it would be better to stay with one convention - with or without `{}`